features/box-decoration-break.yml

```
@@ -0,0 +1,8 @@
+name: box-decoration-break
+description: The `box-decoration-break` CSS property sets whether the box decoration of an element divided across a page, column, or region wraps each fragment or is split across the break.
```

suggestions to include an example of a box decoration and to improve parallelism ("wraps or… splits" instead of "wraps… or is split"):

```suggestion
description: The `box-decoration-break` CSS property sets whether box decorations, such as borders or backgrounds, of an element divided across a page, column, or region wraps each fragment or splits across the break.
```